configure.ac

```diff
@@ -3772,6 +3772,9 @@ esac
 # of __STDC_IEC_559_COMPLEX__.
 AC_RUN_IFELSE([AC_LANG_SOURCE([[
 #include <complex.h>
+#ifndef FFI_TYPE_COMPLEX
```

Comment:
Don't younhave to include ffi.h for that?

Reply:
Yep, but that part was reverted.  The struct module has no dependence on libffi, so it does make sense to have this check independent on libffi too.